packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/machine.test.ts

```diff
@@ -381,47 +389,14 @@ describe('Liveness Machine', () => {
 
   describe('notRecording', () => {
     it('should reach recording state on START_RECORDING', async () => {
-      await transitionToInitializeLivenessStream(service);
+      await transitionToNotRecording(service);
       await flushPromises(); // notRecording: 'waitForSessionInfo'
 
-      service.send({
-        type: 'SET_SESSION_INFO',
-        data: {
-          sessionInfo: mockSessionInformation,
-        },
-      });
-      jest.advanceTimersToNextTimer(); // "recording": "ovalDrawing",
-
-      expect(service.state.value).toEqual({ recording: 'ovalDrawing' });
+      expect(service.state.value).toEqual('start');
     });
   });
 
   describe('recording', () => {
-    it('should reach ovalDrawing state as initial state and respect ovalDrawingTimeout', async () => {
```

Comment:
Should any of this test be in a new place? I didn't see it move.

Reply:
I believe it was covered by another test and was not necesary